stylesheets/components/_alert.scss

```diff
@@ -0,0 +1,109 @@
+// -------------------------------------------
+// This file contains styles related to alerts
+// -------------------------------------------
+
+/**
+* Mixin for alert colors. Accepts arguments for background color, border color and text color.
+**/
+@mixin alert-color($background-color, $border-color, $color) {
+  background-color: $background-color;
+  border-color: $border-color;
+  color: $color;
+}
+
+/**
+* Base alert class.
+**/
+.alert {
+  border: 1px solid;
+  margin: 10px 0;
+  padding: 20px;
+}
+
+/**
+* Wrapper for the alert icon.
+**/
+.alert__icon-wrapper {
+  display: inline-block;
+  vertical-align: middle;
+}
+
+/**
+* Wrapper for text in the alert.
+**/
+.alert__text-wrapper {
+  display: inline-block;
+  vertical-align: middle;
+}
+
+/**
+* Button which closes the alert.
+**/
+.alert__button {
+  appearance: none;
+  background: none;
+  border: 0;
+  color: inherit;
+  float: right;
+  font-family: $sans-serif-stack;
```

Comment:
I don't think you need font styles here if you use a material icon for this instead of "X".